lib/exceptional_synchrony/limited_work_queue.rb

```diff
@@ -43,7 +50,8 @@ def work!
         end.resume
       end
     end
-
+    \
```

Comment:
That backslash just combined those 2 lines into one.  So no effect on Ruby syntax.